ui/lib/core/addon/templates/components/shamir-flow.hbs

```diff
@@ -138,19 +138,20 @@
       <div class="box is-shadowless is-marginless no-padding-top is-fullwidth" data-test-form-text>
         {{#if this.otp}}
           <p>
-            <AlertBanner
-              @type="info"
-              @message="Below is the generated OTP. This will be used to encode the generated Operation Token. Make sure to save this, as you will need it later to decode the Operation Token."
-            />
+            <Hds::Alert @type="inline" @color="highlight" class="has-bottom-margin-s" as |A|>
+              <A.Title>Info</A.Title>
+              <A.Description>
+                Below is the generated OTP. This will be used to encode the generated Operation Token. Make sure to save
+                this, as you will need it later to decode the Operation Token.
+              </A.Description>
+            </Hds::Alert>
           </p>
-          <div class="message is-list has-copy-button" tabindex="-1">
+          <div class="has-background-gray-100 box has-copy-button" tabindex="-1">
```

Comment:
`class="message"` defaults to the old structure's `info` blue background, so this looked a little incohesive beside the new alert banner: 
### before

![image](https://github.com/hashicorp/vault/assets/68122737/4c8ba041-f110-49ae-b037-05cb00005cff)

### after: 
updated to generic grey and removed the erroneous div for now. design has a ticket to revisit this 
<img width="1378" alt="Screenshot 2023-06-15 at 3 41 00 PM" src="https://github.com/hashicorp/vault/assets/68122737/310e002c-41a0-4717-b920-14d6adb51ca0">